client/src/keybind.c

```diff
@@ -168,6 +189,8 @@ void keybind_spiceRegister(void)
 #if ENABLE_AUDIO
     if (audio_supportsRecord())
     {
+      app_registerKeybind(KEY_A, bind_toggleMicDefault, NULL,
+          "Toggle audio recording default");
```

Comment:
```suggestion
          "Cycle audio recording default");
```
I think this is more accurate.

Reply:
Fixed.